app/helpers/admin/claims_helper.rb

```diff
@@ -23,5 +23,10 @@ def admin_personal_details(claim)
         [t("questions.admin.email_address"), claim.email_address],
       ]
     end
+
+    def school_with_link(school)
```

Comment:
`school_with_link` feels like it's going to return the school, with a link. Perhaps `link_to_school` would be more intention revealing?